### DISCUSSION TITLE
Documentation contradicts content

### DISCUSSION THREAD

**Discussion by dartess** [gpbl/react-day-picker#2354] [Support]
I noticed a few issues when trying to update to version 9. They are manageable, but at a certain point I gave up and abandoned it.

From docs https://daypicker.dev/docs/styling

```
--rdp-day-height The height of the day cells.
--rdp-day_button-height The height of the day cells.
```

**Do** two variables represent the same thing?

And from source:

```
--rdp-day_button-height: var(--rdp-day-width);
```

What does this even mean, why is the **width** of the "day cell" equal to the **height** of the "day cell"?

**Comment by gpbl** [comment]
@dartess thanks for the feedback! The documentation for the styling need some updates.

To answer your questions:

- a day cell may contain a button. A day cell is styled via classes with prefix `rdp-day`, while buttons with prefix `rdp-day_button`.
- the width is equal to the height by design.

Please checkout the generated HTML to understand how the classes are applied.
